src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -91,84 +93,44 @@ export const PositionsTriggersCell = ({
     );
   };
 
-  const viewOrdersButton = (
-    <$ActionButton
-      action={ButtonAction.Navigation}
-      size={ButtonSize.XSmall}
-      onClick={onViewOrders ?? undefined}
-      disabled={isDisabled}
-    >
-      {stringGetter({ key: STRING_KEYS.VIEW_ORDERS })}
-      <Icon iconName={IconName.Arrow} tw="stroke-2" />
-    </$ActionButton>
+  const viewOrdersButton = (align: 'left' | 'right') => (
+    <$Container $align={align}>
+      <$ViewOrdersButton
+        action={ButtonAction.Navigation}
+        size={ButtonSize.XSmall}
+        onClick={onViewOrders ?? undefined}
+        disabled={isDisabled}
+      >
+        <$Label>{stringGetter({ key: STRING_KEYS.VIEW_ORDERS })}</$Label>
+      </$ViewOrdersButton>
+    </$Container>
   );
 
-  const renderOutput = ({ label, orders }: { label: string; orders: SubaccountOrder[] }) => {
-    const triggerLabel = ({
-      liquidationWarningSide,
-    }: {
-      liquidationWarningSide?: Nullable<AbacusPositionSides>;
-    } = {}) => {
-      let triggerButtonState = TriggerButtonState.Default;
-
-      if (liquidationWarningSide != null) {
-        triggerButtonState = TriggerButtonState.Warning;
-      } else if (orders.length > 0) {
-        triggerButtonState = TriggerButtonState.HasOrders;
-      }
-
-      const triggerButton = (
-        <$TriggerButton
-          action={ButtonAction.Primary}
-          onClick={openTriggersDialog}
-          triggerButtonState={triggerButtonState}
-          disabled={isDisabled}
-        >
-          {label}
-        </$TriggerButton>
-      );
-      return liquidationWarningSide ? (
-        <WithHovercard
-          align="start"
-          side="left"
-          hovercard={
-            liquidationWarningSide === AbacusPositionSide.LONG
-              ? 'liquidation-warning-long'
-              : 'liquidation-warning-short'
-          }
-          slotButton={
-            <Button
-              action={ButtonAction.Primary}
-              size={ButtonSize.Small}
-              disabled={isDisabled}
-              onClick={openTriggersDialog}
-            >
-              {stringGetter({ key: STRING_KEYS.EDIT_STOP_LOSS })}
-            </Button>
-          }
-          slotTrigger={triggerButton}
-        />
-      ) : (
-        triggerButton
-      );
-    };
+  const editButton = (
```

Comment:
suggestion -- can we actually make the whole cell clickable to open the triggers dialog? (esp. when empty / no view-orders button); currently the target of the edit button seems kinda small and i didn't click it on my first try

Reply:
ah ok will follow up to add in a seperate PR!! gonna merge ahead into parent branch tho for sake of not bothering for re-review